osu.Game/Storyboards/CommandTimelineGroup.cs

```diff
@@ -45,11 +45,30 @@ public CommandTimelineGroup()
             };
         }
 
+        /// <summary>
+        /// Returns the earliest visible time. Will be null unless this group has an <see cref="Alpha"/> command with a start value of zero.
```

Comment:
```suggestion
        /// Returns the earliest visible time. Will be null unless this group's first <see cref="Alpha"/> command has a start value of zero.
```

subtle but important (I think?) distinction